templates/security/esm.html

```diff
@@ -10,632 +10,424 @@
-            <td>amd64, arm64, s390x, ppc64el, RISC-V</td>
-          </tr>
-        </tbody>
-      </table>
+    <div class="u-fixed-width">
+      <div class="p-image-container--cinematic is-cover">
+        {{ image(url="https://assets.ubuntu.com/v1/a7dc8f0d-benefits-of-ubuntu-pro.png",
+                alt="",
+                width="2464",
+                height="1028",
+                hi_def=True,
+                loading="lazy",
+                attrs={"class": "p-image-container__image"},) | safe
+        }}
+      </div>
     </div>
   </section>
 
-  <section class="p-strip is-deep u-no-padding--top">
-    <div class="row">
+  <section class="p-section">
+    <div class="row--50-50">
       <hr class="p-rule" />
-      <div class="col-6">
-        <h2>
-          ESM on public clouds
-          <br class="u-hide--small u-hide--medium" />
-          with Ubuntu Pro
-        </h2>
+      <div class="col">
+        <div class="p-section--shallow">
+          <h2>
+            The difference between
+            <br />
+            'esm-apps' and 'esm-infra'
+          </h2>
+        </div>
       </div>
-      <div class="col-6">
+      <div class="col">
+        <p>
+          Ubuntu Pro is a broad subscription that includes many different variations of open source packages to meet different needs. For example, Pro includes a set of package versions that are compliant with FIPS regulations. You would want these versions only on machines that need to meet FIPS requirements, so you can choose to enable that stream specifically on those machines.
+        </p>
         <p>
-          If you are running Ubuntu 18.04 LTS images on the public cloud and are looking for continued security coverage with ESM, it is recommended to launch new, Ubuntu Pro images for <a href="/azure/pro">Azure</a>, <a href="/aws/pro">AWS</a> and <a href="/gcp/pro">Google Cloud</a>. Ubuntu Pro images are paid, premium images that are optimised and priced for the cloud, with security and compliance features built-in.
+          There are two streams which cover broad-based package security updates; we label these “apps” and “infra”. The “esm-apps” stream covers all 'Universe' packages for ten years from the release of the LTS.
         </p>
         <p>
-          Learn more about Ubuntu Pro on <a href="/azure/pro">Azure</a> | <a href="/aws/pro">AWS</a> | <a href="/gcp/pro">Google Cloud</a>
+          The “esm-infra” stream covers 'Main' packages for the period after the standard five year security maintenance of 'Main' packages ends. We call this 'infra' because it is commonly used to build our private cloud, storage and kubernetes clusters, where 'Universe' packages are not typically deployed. You can get a lower-cost Ubuntu Pro (infra-only) subscription if you only want the infra components, which equates to our original ESM offering.
         </p>
+        <div class="p-cta-block">
+          <a href="/security/esm#get-in-touch">Get in touch&nbsp;&rsaquo;</a>
```

Comment:
in case JS is disabled this link won't open a modal. We need a fallback that will still open a Contact Us static page in case JS is disabled.